lib/resources/r4/encounter_patch.yaml

```diff
@@ -64,3 +64,36 @@ operations:
         <li>`Encounter.type` value should be a list.</li>
         <li>When a list is provided, the first value would be used to patch `Encounter.type`.</li>
       </ul>
+
+  - name: replace-extension
+    path: /extension
+    operation: replace
+    type: List of Extension
+    url: http://hl7.org/fhir/encounter-definitions.html#Encounter.extension
+    description: The aray of extension of encounter.
+    example: |
+      {
+        "op": "replace",
+        "path": "/extension",
+        "value":  [
+          {
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status",
+            "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003489",
+                "code": "40016441",
+                "display": "1st Attempt - Left Message",
+                "userSelected": true
+              }
+            ],
+            "text": "1st Attempt - Left Message"
+           }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>`Encounter.extension` value should be a list.</li>
```

Comment:
Encounter.extension makes no sense here.

Reply:
It looks like this is the pattern followed.

Reply:
I don't think this is needed at all. The field is /extension. By the fhir spec it is defined as a list already.

Reply:
isnt type too? but its noted there....

Reply:
True... I don't think it's needed there either, but let's make the change for this right now since this is new documentation.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/a9a65e45d04546e76fa09cc9d8b4ba7395d2f7d2#diff-97d84edc87e609cdbf80d6ad20cbc681R96